drivers/usb/device/usb_dc_native_posix.c

```diff
@@ -340,7 +333,7 @@ int usb_dc_ep_write(const uint8_t ep, const uint8_t *const data,
 	}
 
 	/* Check if IN ep */
-	if (USBIP_EP_ADDR2DIR(ep) != USB_EP_DIR_IN) {
+	if (USB_EP_GET_DIR(ep) != USB_EP_DIR_IN) {
```

Comment:
``` diff
-	/* Check if IN ep */
-	if (USBIP_EP_ADDR2DIR(ep) != USB_EP_DIR_IN) {
+	/* Check if OUT ep */
+	if (USB_EP_DIR_IS_OUT(ep)) {
```
?

Reply:
No, I will not change this logic here. Also not relevant to this PR.